PDF support for file upload from embedded URL (#197)
directus/docs: api/reference.md

Directus supports adding embed videos from YouTube and Vimeo to Directus Files using the video URL. Saving an image as its thumbnail.

> Support of uploading image and pdf from URL to Directus Files is also there.

Reviewer: ```suggestion
::: tip
You can also import an image or PDF from a URL.
:::
```
